invenio_rdm_records/services/stats/permissions.py

```diff
@@ -47,11 +43,12 @@ def can(self):
         )
 
 
-def default_deny_permission_factory(query_name, params):
-    """Default deny permission factory.
+def permissions_policy_lookup_factory(query_name, params):
```

Comment:
Actually, the name has been inaccurate ever since I've built in the permission translation class, sorry about that! :sweat_smile: